internal/topic/topicwriterinternal/writer_config.go

```diff
@@ -22,5 +21,4 @@ type WritersCommonConfig struct {
 	cred               credentials.Credentials
 	credUpdateInterval time.Duration
 	clock              clockwork.Clock
-	forceCodec         rawtopiccommon.Codec
```

Comment:
This line was removed because the structcheck linter wrote `internal/topic/topicwriterinternal/writer_config.go:25:2: "forceCodec" is unused (structcheck)
        forceCodec         rawtopiccommon.Codec
`